Jenkinsfile

```diff
@@ -104,14 +104,16 @@ spec:
 							dir ('eclipse.platform.swt.binaries') {
 								sh '''
 									/opt/tools/apache-maven/latest/bin/mvn install \
-										--batch-mode -Pbuild-individual-bundles -DforceContextQualifier=zzz -Dnative=gtk.linux.x86_64 \
+										--batch-mode --no-transfer-progress
```

Comment:
Missing `\` at end of line.

Reply:
🤦🏻‍♂️ Thanks @mickaelistria . That happens when you try to change something online via the browser..